conda/cli/main_info.py

```diff
@@ -331,8 +331,13 @@ def execute(args, parser):
         return
 
     if args.packages:
-        print_package_info(args.packages)
-        return
+        from ..resolve import ResolvePackageNotFound
+        try:
+            print_package_info(args.packages)
+            return
+        except ResolvePackageNotFound as e:
```

Comment:
Nice PR.  Just make this line

    except ResolvePackageNotFound as e:  # pragma: no cover